vocab/properties.json

```diff
@@ -865,7 +853,7 @@
         ]
     },
     "homepage": {
-        "description": null,
+        "description": "Central website of something or someone.",
```

Comment:
Maybe "Principal" or "Main" rather than "Central"?

Reply:
Whatever you think is better ;). If you don't object I'll use "main"